### PR TITLE
feat(platforms): Add FastAPI and Starlette as platforms

### PR DESCRIPTION
**Pull request by antonpirker**
Adds the newly supported FastAPI and Starlette Python Frameworks to the platform selector in "Create Project".

### CODE REVIEW

**Comment by andriisoldatenko** [comment]
@antonpirker i rerun failing test, i think it's just flaky. 
For: Visual Snapshot  - i think you need to rebase from master.

**Comment by antonpirker** [comment]
Thanks for checking @andriisoldatenko ! Rebased, lets hope for the best :-)

File: fixtures/integration-docs/python-starlette.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "html": "<div class=\"section\" id=\"installation\"></div>\n",
+    "link": "https://docs.getsentry.com/clients/python/integrations/starlette/",
```
[the remark this answers is not in the document]
Comment (antonpirker) [reply fixtures/integration-docs/python-starlette.json:3]: ok, thanks for the hint!

**Comment by antonpirker** [comment]
Closing this in favor of two seperate PRs:

Backend: https://github.com/getsentry/sentry/pull/37099
Frontend: https://github.com/getsentry/sentry/pull/37100

**Pull request closed without merging**